### PR TITLE
frmw1214-rxoptiontable

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: https://jira.rax.io/browse/FRMW-1214

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM

### CODE REVIEW

File: src/elements/Forms/styles/rxForm.less [hunk 1]
```diff
@@ -217,12 +217,7 @@ rx-field-name {
               color: @app-disabled-text-color;
             }
           }
-
-          .rx-option-table {
-            table-layout: fixed;
-            width: 100%;
-          }
```
Comment (CITguy) [inline src/elements/Forms/styles/rxForm.less]: We'll need this applied to Layout 3 in the demo. This is what's causing the visreg changes.

File: demo/less/_shame.less [hunk 2]
```diff
@@ -1,14 +1,3 @@
-.rx-option-table {
-  thead {
-    tr {
-      th:first-child {
-        /* Fix column width in demos */
-        width: 40px;
```
Comment (CITguy) [inline demo/less/_shame.less]: Note: style moved to `demo/less/app.less` for demo use only.

File: demo/templates/styles/layouts/form-page.html [hunk 3]
```diff
@@ -52,27 +52,39 @@ <h2 class="subdued">Sample Create Page</h2>
...
+                        <th>Name</th>
+                        <th>Static Content</th>
+                        <th>Expression 2</th>
+                        <th>Expression 3</th>
+                        <th>Expression 4</th>
+                      </tr>
+                    </thead>
+                    <tbody>
+                      <tr ng-repeat="datum in radioTableData">
+                        <td><rx-input><input rx-radio name="radiooption"></rx-input></td>
```
Comment (CITguy) [inline demo/templates/styles/layouts/form-page.html]: `<rx-input>` not necessary. Please remove `<rx-input>`.

```html
<td><input rx-radio name="radiooption" /></td>
```

**Comment by cynprudence** [comment]
Video LGTM

**Pull request merged**